### ISSUE TITLE
Barcode height

### ISSUE THREAD

**Issue by nicknomo** [graphicore/librebarcode#49]
Would it possible to adjust the relative height of the barcode lines?  Any pointers on how?  I'd really like to generate barcodes that look more proportional to a UPC code.  I can to fork this to make changes, just not sure where to start.

**Comment by graphicore** [comment]
Forking would certainly be the quickest way. You should start looking at https://github.com/graphicore/librebarcode/blob/master/app/bin/buildAll and modify values around here:

https://github.com/graphicore/librebarcode/blob/f9864c42b2c467f255659c8851c124e4cd56c67a/app/bin/buildAll#L73-L79

Also, ideally, especially if you are going to release change these to something else:
```
    local FAMILY_BASE_NAME="Libre Barcode"
    local FILE_BASENAME="LibreBarcode"
    local VENDOR_ID="GRCR"
# and add to
    local COPYRIGHT="Copyright 2017-2020 The $FAMILY_BASE_NAME Project Authors (https://github.com/graphicore/librebarcode)"
```

To avoid confusion, both of users and of the Operating System font menus.

At some point in the future I'll add variable font tables to these fonts, height could be a slider in the fonts and static instances could be made from those automatically. But it won't happen too soon.

**Comment by graphicore** [comment]
Labeled as enhancement, as a reminder of the variable font idea.

**Comment by wajeemba** [comment]
I totally agree with nicknomo. Would absolutely love a way to adjust the height of the Code 39 text in particular!

**Comment by TheMDev** [comment]
I ended up trying the recommendation by @graphicore 

> You should start looking at https://github.com/graphicore/librebarcode/blob/master/app/bin/buildAll and modify values around here:
https://github.com/graphicore/librebarcode/blob/e44a85e71bd40564fbd1e88b215c619f6be85858/app/bin/buildAll#L73-L79

However for some reason the variables are always overridden by the defaults in: 
https://github.com/graphicore/librebarcode/blob/e44a85e71bd40564fbd1e88b215c619f6be85858/app/lib/builder/abstract.js#L531-L537
